### PR TITLE
Running bundles now continuously heartbeat the server during dependency/image downlaods

### PR DESCRIPTION
**Pull request by fabeschan**

### CODE REVIEW

File: worker/codalabworker/run.py [hunk 1]
```diff
@@ -127,14 +127,7 @@ def resume(self):
         if not self._bundle_service.resume_bundle(self._worker.id, self._uuid,
                                                  start_message):
             return False
-
-        # Start a thread for this run.
-        def resume_run(self):
```
Comment (percyliang) [inline worker/codalabworker/run.py:132]: How come we don't need a thread anymore?  That seems orthogonal sending a heartbeat?

Comment (fabeschan) [reply worker/codalabworker/run.py:132]: Ya so the reason for that is this thread that runs `resume` only starts after dependencies and images are downloaded, when `resume` should be repeatedly called during those phases. So we remove this whole thread set up and put `resume` in `_throttled_updater` which already runs periodically on a thread.

**Review by percyliang — APPROVED** [review]

File: worker/codalabworker/run.py (hunk 1 above)
Comment (fabeschan) [reply worker/codalabworker/run.py:132]: Turns out we do need a thread.. whoops
